vizro-core/src/vizro/models/_page.py

```diff
@@ -45,6 +46,7 @@ class Page(VizroBaseModel):
 
     components: List[ComponentType]
     title: str = Field(..., description="Title to be displayed.")
+    description: Optional[str] = Field(None, description="Page description for meta tags.")
```

Comment:
```suggestion
    description: str = Field("", description="Page description for meta tags.")
```

The handling of `Optional` and `None` default values with pydantic can be extremely confusing and something we've discussed on the team many times to try to come up with the best solution (most recently https://github.com/mckinsey/vizro/pull/210) 😅 I won't bore you with the details and honestly it's not worth trying to understand any of it, but this one at least is a simple case: if you give the field a default value then it will already be optional in the sense that it's not a required field. Adding `Optional` and `None` in this case makes things a bit harder for type-checking etc. so we should just use empty string instead.

Reply:
Good to know (I haven't used pydantic much) - I'll also remove the `Optional` from the imports.

Reply:
Perfect 👌 

Just unresolving the conversation to point out that our linter will automatically do things like remove unused imports for you - generally I just make all my changes and then run `hatch run lint` at the end without needing to worry too much about having perfectly linted code in intermediate commits.